UnityProject/Assets/Scripts/Core/Chat/EmoteActionManager.cs

```diff
@@ -10,6 +12,45 @@ public class EmoteActionManager : SingletonManager<EmoteActionManager>
 		[SerializeField] private EmoteListSO emoteList;
 		public EmoteListSO EmoteList => emoteList;
 
+		private KeybindManager.KeyCombo windowInputComboKeys = new KeybindManager.KeyCombo();
```

Comment:
you keep adding hardcoded keys for new features. Can't we use the input system we already have?

Reply:
???
The keybind manager checks for actions based on keybinds the users set